swagger.yaml

```diff
@@ -73,3 +85,54 @@ definitions:
       details:
         type: string
         description: A link to further information about the error.
+
+  ResultObject:
+    properties:
+      links:
+        $ref: "#/definitions/Links"
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/ResourceObject"
+
+  ResourceObject:
+    properties:
+      id:
+        type: string
+      type:
+        type: string
+      attributes:
+        $ref: "#/definitions/Attributes"
+      links:
+        $ref: "#/definitions/SelfLink"
+
+  Links:
+    properties:
+      self:
+        type: string
+      first:
+        type: string
+      last:
+        type: string
+      prev:
+        type: string
+      next:
+        type: string
+
+  SelfLink:
+    properties:
+      self:
+        type: string
+        description: Url to this resource / location.
+
+  Attributes:
+    properties:
+      name:
+        type: string
+        description: Name of the department usually referred to as orgDescription.
+      bcName:
+        type: string
+        description: Name of the business center
+      organizationCode:
+        type: string
+        description: numeric orgCode for the department
```

Comment:
This might be nitpicking, but could each "description" in this object start with a capital letter and end with a period?